### DISCUSSION TITLE
How to Move Table Pagination to Top-Right Corner

### DISCUSSION THREAD

**Discussion by susansiow** [quasarframework/quasar#8964] [General - Components / Directives / etc]
How can I move the pagination to the top-right corner with v-slot? I was not able to obtain a solution from the documentation - https://quasar.dev/vue-components/table.

```
 <template v-slot:top-left>
       ?
 </template>
```
Kindly assist, thank you!

**Comment by metalsadman** [comment]
you can do something like this https://codepen.io/metalsadman/pen/WNRapWe?editors=1010, tho it's better to have a custom pagination since the `top-left` slot already exposes the pagination object, the problem is you will have to write the template yourself to mimic the bottom pagination one.

**Reply by susansiow** [reply]
Hi @metalsadman 

Thanks for the reply, I have a question, what does it mean by "it's better to have a custom pagination since the top-left slot already exposes the pagination object"? Do you mean that this will cause duplication, thus not ideal?

**Comment by lisabonji** [comment]
The layout of your website is great. I would love to be your active subscriber always.
I loved your content pattern and modulation and would suggest others too, visit your website.
[ cernos gel](
https://www.pharmamedsstore.com/product/cernos-gel-1/)
